src/convertToObject.js

```diff
@@ -3,16 +3,29 @@
 /**
  * Implement convertToObject function:
  *
- * Function takes a string with styles (see an example in [stylesString.js](./stylesString.js))
+ * Function takes a string with styles
+ * (see an example in [stylesString.js](./stylesString.js))
  * and returns an object where CSS properties are keys
- * and values are the values of related CSS properties (see an exampl in [test file](./convertToObject.test.js))
+ * and values are the values of related CSS
+ * properties (see an exampl in [test file](./convertToObject.test.js))
  *
  * @param {string} sourceString
  *
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const StylesObject = {};
+
+  sourceString.split(';')
+    .map(style =>
+      style.trim()).filter(style => style)
+    .forEach(style => {
+      const styleValue = style.split(':');
+
+      StylesObject[styleValue[0].trim()] = styleValue[1].trim();
+    });
```

Comment:
don't use forEach to mutate anything, forEach is needed to iterate through array elements and use callback for every of them. Use reduce to build an object
Also move callback outside of function to avoid creating it everytime